code/modules/aberrants/organs/machinery/regurgitator.dm

```diff
@@ -163,7 +161,7 @@
 
 	current_tick += 1
 
-	while(biomatter_counter > 59)
+	while(biomatter_counter > 59.99)
```

Comment:
If you want to ensure biomatter does not go negative due to rounding issue, you can also put 
`biomatter_counter = max(0, biomatter_counter - 60)		// Flesh cubes have 60 biomatter` in `spit()`